language/nodejs/build-images.md

```diff
@@ -77,6 +77,8 @@ Switch back to the terminal where our server is running. You should now see the
 2020-XX-31T16:35:21:3560  INFO: GET /test
 ```
 
+At this stage, you're done with testing this server script locally, so stop it running by going back to the first terminal session where you started it up, and press ctrl-c.
```

Comment:
Thanks! Yes, it makes sense to be sure it's stopped.

Perhaps we should show what it looks like after pressing that, and guide into the next part. Something like;

    Great! We verified the application works. At this stage, you're done with testing
    the server script locally.
    
    Press `CTRL-c` from within the terminal session where the server is running to
    stop it.
    
    ```console
    2021-08-06T12:11:33:8930  INFO: POST /test
    2021-08-06T12:11:41:5860  INFO: GET /test
    ^Cshutting down...
    ```
    
    We will now continue to build and run the application in Docker.


(sorry, couldn't make the a GitHub "suggestion", as it didn't like the nested "fences" (```)). Happy to push those changes to your branch though.

Reply:
/cc @usha-mandya

Reply:
I've added your suggestions to the PR 👍